welcome.md

```diff
@@ -0,0 +1,37 @@
+---
+
+title: Welcome to UMA
+sidebar_label: Welcome to UMA
+
+---
+
+UMA stands for "Universal Market Access" 
+
+Our mission is to make financial markets universally fair and accessible.
+
+UMA is driven by a shared belief that financial markets should be universally accessible - unrestrained by censorship, pre-existing financial and social capital, or beliefs and values. The economic freedom created by open market access enables everyone equal access and footing to pursue prosperity and build their financial independence.  Free and universal access to financial markets is a pre-requisite to a free, open and fair financial system. 
+
+UMA's goal is to enable anyone to attain or transfer any form of risk seamlessly and securely through the UMA protocol thus empowering everyone to participate in a universally accessible financial system.
+
+## What is UMA ##
+
+UMA builds open source infrastructure for developers to create financial instruments in a permissionless, secure and efficient manner. 
+
+This infrastructure includes:
+ - [Priceless financial contract templates](getting-started/priceless-financial-contracts.md) which can be used to create synthetic tokens
+ - An optimistic oracle known as the [Data Verification Mechanism](getting-started/oracle.md) (DVM) to enforce financial contracts built on UMA
+ - A [Decentralized Governance Process](getting-started/governance.md) to manages improvements of UMA protocol
+
+These components enable the creation of fast, efficient, and secure synthetic assets on Ethereum.

```

Comment:
It's the first time we call out synthetic assets. We should link to the section that discusses this in more detail.

Reply:
I see we removed the header for "understanding UMA" for someone who is looking to learn more about uma where should they head?